### DISCUSSION TITLE
Shrines' effect description?

### DISCUSSION THREAD

**Discussion by Darmorque** [diasurgical/DevilutionX#7339] [Ideas]
I't quite simple and obvious idea. Would that be possible to add the actual effect desription of every shrine just beneath its name that is displayed in the middle of interface anyway?

It could be optional for those who dislike it, but honestly - I'm never 100% certain which shrine does what, and even despite existing option to disable the ability of clicking 'destructive' shrines, most of the time I still Alt-Tab the game to check those shrines descriptions just to be sure what's gonna happen. It's not a big deal, but in fact that's the only thing that makes me jump out and back into the game, and therefore it would be awesome to have this tiny info included in the game.

**Comment by qndel** [comment]
https://github.com/diasurgical/devilutionX/issues/7033
